migrations/v2.js

```diff
@@ -0,0 +1,205 @@
+import { describe, whereContent, whereFromPlugin, mutateContent, checkContent, updatePlugin } from 'adapt-migrations';
+
+describe('adapt-contrib-accordion - v1.0.0 > v2.0.0', async () => {
+  let course, courseAccordionGlobals, accordions;
+
+  whereFromPlugin('adapt-contrib-accordion - from v1', { name: 'adapt-contrib-accordion', version: '<=2.0.0' });
+
+  whereContent('adapt-contrib-accordion - where accordion', async content => {
+    accordions = content.filter(({ _component }) => _component === 'accordion');
+    if (accordions) return true;
+  });
+
+  /**
+   * * Add JSON field to component and set attribute.
+   */
+  mutateContent('adapt-contrib-accordion - add accordion._supportedLayout', async () => {
+    accordions.forEach(accordion => {
+      accordion._supportedLayout = 'half-width';
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check accordion._supportedLayout attribute', async () => {
+    const isValid = accordions.every(({ _supportedLayout }) => _supportedLayout !== undefined);
+    if (!isValid) throw new Error('adapt-contrib-accordion - _supportedLayout not added to every instance of accordion');
+    return true;
+  });
+
+  /**
+   * * Add JSON field to component and set blank.
+   */
+  mutateContent('adapt-contrib-accordion - add accordion.instruction', async () => {
+    accordions.forEach(accordion => {
+      accordion.instruction = '';
+    });
+    return true;
+  });
+
+  checkContent('adapt-contrib-accordion - check accordion.instruction atrribute', async () => {
+    const isValid = accordions.every(({ instruction }) => instruction !== undefined);
```

Comment:
This can be shortened for readability (a few examples of throughout), but still working nicely. 

```suggestion
    const isValid = accordions.every(({ instruction }) => instruction);
```